src/lt/objs/editor.cljs

```diff
@@ -125,23 +125,23 @@
   [e]
   (. (->cm-ed e) (getValue)))
 
-(defn ->token [e pos]
+(defn- ->token [e pos]
```

Comment:
I'm pretty sure this is used quite a bit (ref clojure plugin, and I have used it alot)  and needs to be public. Or perpaps there is a better way to get hold of the (cm) token at a given pos ?